### DISCUSSION TITLE
Thrustmaster Airbus throttle quadrant settings

### DISCUSSION THREAD

**Discussion by Riggie** [flybywiresim/aircraft#3706] [Help] [answered]
In the section regarding the Throttle configuration docs for the quadrant,  am I correct in thinking that these configurations are for use with the stable version of the A32NX? ........ info found here https://github.com/flybywiresim/a32nx/tree/fbw/docs

Also, am I correct in thinking that the ThrottleConfiguration.ini file found in the a32nx Work folder is the Microsoft settings for their Neo?  and is it only created because of the FBW Neo installation?
Should there not be a FBW ThrottleConfiguration.ini?
I am struggling to get the DetentFlexMct detent to line up,  all the rest are spot on.  Saying that the power ratios are not correct as I am still moving the sensitivity sliders just to create an idle position.
I am working on the settings that idle is 20%,  CLB 80% Flex 90% Toga 100%  are these correct?

I have the Thrustmaster latest software 1.61 installed but to tell you the truth I have had better results without it!

Has anyone managed to set the Quadrant set up correctly?

Any help will be appreciated

cheers
Riggie

**Comment by Mickepilot** [comment, marked as answer]
Hello "Riggie", maybe this is a help for you?:
Please try the following steps:

1) Reset the MS FS A320 TCA Quadrant control "Sensitivity" to "0" values (=linear response)
2) Do a new calibration of the Thrustmaster Airbus TCA Quadrant, pressing both L & R red side buttons in all 5 positions (=incl. full reverse position!)

Using the A32NX "Custom FBW", look for this "hidden" file: "ThrottleConfiguration.ini" under:
"C:\Users\ -the PC owner- \AppData\Local\Packages\Microsoft.FlightSimulator_8wekyb3d8bbwe\LocalState\packages\a32nx\work"

3) Enter these slightly different values in the "ThrottleConfiguration.ini" file:

Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.70
DetentIdle = -0.44
DetentClimb = -0.10
DetentFlexMct = 0.53
DetentTakeOffGoAround = 1.00 

Now, I got the Thrustmaster Airbus TCA Quadrant, to correctly match the detents on both the throttle quadrant and the simulator!

**Reply by Riggie** [reply]
Hi Mickepilot, Thanks for the info
I used all your settings for the  "ThrottleConfiguration.ini" file:  and I now have  60% reverse thrust.
I had to make one adjustment in the Sensitivity section to correct only one detent. ( see attached )
The power range at Climb, FLX, Toga, are around 88, 92, 95%, and idle is 19.7% so I am happy about that, it will do me.
thanks for your help
cheers
Riggie

![Reverse Thrust](https://user-images.githubusercontent.com/77942251/109430592-0e156480-79fa-11eb-88d7-855f09334437.jpg)
